src/main/java/com/frame/core/domain/post/controller/PostController.java

```diff
@@ -35,6 +35,12 @@ public GetPostDetailResponse getPostDetail(@RequestParam Long postId) {
         return postService.getPostDetail(postId);
     }
 
+    @DeleteMapping("/detail")
+    @ResponseStatus(HttpStatus.OK)
+    public void deletePostDetail(@RequestParam Long postId) {
```

Comment:
delete 할 때는 detail 이렇게 필요 없음. + Request Param으로 받으면 안됨 Body로 받아야함
```suggestion
    public void deletePostDetail(@RequestBody @Valid DeletePostRequest request) {
```

Reply:
아하